tool/tctl/common/usage.go

```diff
@@ -45,7 +45,7 @@ Examples:
   > tctl nodes add
 
   Generates a token when can be used to add a regular SSH node to the cluster.
-  The token genrated single-use token will be valid for 30 minutes.
+  The generated token will be valid for 30 minutes.
```

Comment:
While we're here I feel like we should replace this with `tctl tokens add`, since not everything is a "node".

Reply:
This is the description for `tctl help nodes add`. Do we want to reference you could use `tctl tokens add --type=node`?

Reply:
Ah, I see. Ignore me.